src/pal/inc/mbusafecrt.h

```diff
@@ -31,6 +31,12 @@ typedef int errno_t;
 // define the return value for success 
 #define SAFECRT_SUCCESS 0
 
+#if defined(_MSC_VER) || defined(__llvm__)
+#define THROW_DECL
+#else
+#define THROW_DECL throw()
```

Comment:
Why do we need to add throw() to some functions with GCC?

Reply:
GCC has a prototype mismatch.

/usr/include/string.h:43:28: error: declaration of ‘void* memcpy(void*, const void*, size_t) throw ()’ has a different exception specifier
        size_t __n) __THROW __nonnull ((1, 2));

Reply:
ok